api/lib/application/organizations/organization-controller.js

```diff
@@ -249,6 +249,14 @@ module.exports = {
     return h.response(organizationInvitationSerializer.serialize(organizationInvitations)).created();
   },
 
+  async cancelOrganizationInvitation(request, h) {
+    const organizationInvitationId = request.params.invitationId;
+    const cancelledOrganizationInvitation = await usecases.cancelOrganizationInvitation({
+      organizationInvitationId: organizationInvitationId,
+    });
```

Comment:
La repetition est redondante
```suggestion
    const cancelledOrganizationInvitation = await usecases.cancelOrganizationInvitation({ organizationInvitationId });
```